client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +260,14 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
+				<div className="help-contact__placeholder">
+					<HelpContactForm disabled={true}
+						showSubjectField={ false }
```

Comment:
Don't need the `showSubjectField` here since the default value is `false`. See [here](https://github.com/Automattic/wp-calypso/blob/update/olark-loading-screen/client/me/help/help-contact-form/index.jsx#L37) for all the defaults.